bin/cli.js

```diff
@@ -15,6 +15,7 @@ Options
   --help                display this
   --output, -o          Specify output file (default: stdout)
   --auth                (optional) Provide an authentication token for private URL
+  --immutableTypes      (optional) Generates immutable types (readonly properties and readonly array)
```

Comment:
nit: `meow` allows camelCase or kebab-case flags; WDYT about just using kebab-case here? Both should be supported behind-the-scenes

Reply:
oh, yeah, definitely. I didn't even notice `prettier-config` and `raw-schema` being `kebab-case`.